nfcreader/src/main/java/company/tap/nfcreader/open/reader/TapNfcCardReader.java

```diff
@@ -58,7 +72,9 @@ public boolean isSuitableIntent(Intent intent) {
             logger.debug("No TAG in intent");
             return false;
         }
-
+        Map<String, String> parameters = new HashMap<String, String>();
+        parameters.put("intent", String.valueOf(intent));
```

Comment:
What you expect to get as a result of `String.valueOf(intent)`

Reply:
This would just notify us that an attempt has been made to open camera. If the intent is suitable or not for getting image. Just to mark an attempt has been made

Reply:
The key is `intent` and what is the value?

Reply:
Currently this is coming. 
<img width="1120" alt="Screen Shot 2020-04-08 at 4 17 14 PM" src="https://user-images.githubusercontent.com/57221514/78788477-89ba8d00-79b4-11ea-9da2-9696b51c9534.png">

Reply:
So it's not a valid value to be pushed 🤷🏻‍♂️

Reply:
Any suggestions what can it be instead

Reply:
Based on what you need to know 

> If the intent is suitable or not for getting image

You can pass the value True/False `parameters.put("Suitable Intent", isSuitable);`

Reply:
great thank you